library/core/src/cell.rs

```diff
@@ -2032,6 +2032,26 @@ impl<T> UnsafeCell<T> {
 }
 
 impl<T: ?Sized> UnsafeCell<T> {
+    /// Converts from `&mut T` to `&mut UnsafeCell<T>`.
+    ///
+    /// # Examples
+    ///
+    /// ```
```

Comment:
```suggestion
    /// ```
    /// # #![feature(unsafe_cell_from_mut)]
```